Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -94,6 +100,18 @@ IndicatorInfoProvider {
             }
         }
     }
```

Comment:
Nit: new line (can we add this to Swiftlint?)

Reply:
I was following the linter in that one lol

Reply:
🤔 ok :)